primitives/domain/src/lib.rs

```diff
@@ -460,7 +518,7 @@ pub struct CommitteeHash(pub Vec<u8>);
 /// Note: A Registration Transaction is called by a user on Cardano to register themselves as a Sidechain Authority Candidate
 #[derive(Debug, Clone, Encode, Decode, PartialEq, Eq, TypeInfo)]
 #[cfg_attr(feature = "serde", derive(Serialize))]
-pub struct RegistrationData {
+pub struct AdaRegistrationData {
```

Comment:
If its on Cardano, then CardanoRegistrationData. Ada is only a token.

Reply:
I assumed this is only related to ADA staking. In the case we want to add Cardano's native tokens into Minotaur, will they have the same registration data?

Reply:
imo `CardanoRegistrationData` should work for both ada and native tokens, while `AdaRegistrationData` is only for ada